generated_chains_ton.go

```diff
@@ -0,0 +1,19 @@
+// Code generated by go generate please DO NOT EDIT
```

Comment:
Just an observation that the generated code is simpler than the generator which is a c/p of another generator. We should rethink this approach in the future.